src/main/resources/project/ec_setup.pl

```diff
@@ -210,6 +210,46 @@
                  category    => "Deployment"
                 );
 
+my %start_vm = (
+                 label       => "Windows Azure - Start Virtual Machine",
+                 procedure   => "Start VM",
+                 description => "Start Virtual Machine",
+                 category    => "Deployment"
+                );
+
+my %stop_vm = (
+                 label       => "Windows Azure - Stop Virtual Machine",
+                 procedure   => "Stop VM",
+                 description => "Stop Virtual Machine",
+                 category    => "Deployment"
+                );
+
+my %restart_vm = (
+                 label       => "Windows Azure - Restart Virtual Machine",
+                 procedure   => "Restart VM",
+                 description => "Restart Virtual Machine",
+                 category    => "Deployment"
+                );
+
+my %teardown = (
+                 label       => "Windows Azure - TearDown Virtual Machines",
+                 procedure   => "TearDown",
+                 description => "Delete Virtual Machine (Commander Resource/ ResourcePool)",
```

Comment:
Avinash, I don't see this change in the latest PR. Can you make this change so I can merge the PR? 
Thanks,
Urvashi